### DISCUSSION TITLE
Import from moinmoin?

### DISCUSSION THREAD

**Discussion by mjog** [outline/outline#8011] [General]
Hi all, I have a legacy MoinMoin wiki with a fair bit of content. I would like to migrate it to Outline. Is this possible in any automated way?

Thanks!

**Comment by tommoor** [comment]
There are two reasonable options:

- Use the [API](https://www.getoutline.com/developers) to programmatically import your documents
- Export HTML files from MoinMoin and construct a zip file in the same structure as is expected by our Import feature. The best way to match this would be to export an existing Outline collection as HTML to see how it is structured.
